src/core_atmosphere/mpas_atm_core.F

```diff
@@ -262,6 +266,15 @@ function atm_core_init(domain, startTimeStamp) result(ierr)
          call mpas_pool_get_array(state, 'xtime', xtime, 1)
          xtime = startTimeStamp
 
+         ! Set the units to be cf compliant 'seconds since <cf-timestamp>'
+         call mpas_pool_get_field(state, 'Time', Time_field) 
+         call mpas_modify_att(Time_field % attLists(1) % attList, 'units', &
+              'seconds since ' // mpas_string_replace(startTimeStamp, '_', ' '))
```

Comment:
So that we use a consistent reference time for restart runs, I think we'll need to use the `initial_time` variable rather than the start time. For example, if I do a cold-start run and simulate for two days, I get
```
	float Time(Time) ;
		Time:units = "seconds since 2010-10-23 00:00:00" ;
		Time:long_name = "CF convention compliant model valid time" ;
```
with values
```
 Time = 0, 21600, 43200, 64800, 86400, 108000, 129600, 151200, 172800 ;
```
However, if I restart from T+24h, I get
```
	float Time(Time) ;
		Time:units = "seconds since 2010-10-24 00:00:00" ;
		Time:long_name = "CF convention compliant model valid time" ;
```
with times
```
 Time = 21600, 43200, 64800, 86400 ;
```
in the model output. (Note that for a restart, the first history file record isn't written, hence no value of 0 for `Time`).